custom/icds_reports/scripts/README.md

```diff
@@ -0,0 +1,118 @@
+
+    * one row per table to migrate
+    * date of the given table (based on the table name or check constraint)
+    * target table to copy the data into
+    * migrated status
+
+4. Copy the data
+
+    Using the `custom/icds_reports/scripts/migrate_tables.py` script repeatedly top copy the data.
+
+    ```
+    $ python migrate_tables.py --help
+    usage: migrate_tables.py [-h] -D SOURCE_DB -O SOURCE_HOST -U SOURCE_USER -d
+                             TARGET_DB -o TARGET_HOST -u TARGET_USER
+                             [--start-date START_DATE] [--end-date END_DATE]
+                             [--start-after-table START_AFTER_TABLE]
+                             [--table TABLE] [--confirm] [--dry-run]
+                             [--parallel PARALLEL]
+                             db_path
+
+    Migrate DB tables from one DB to another using pg_dump. Compainion to
+    custom/icds_reports/management_commands/generate_migration_tables.py
+
+    positional arguments:
+      db_path               Path to sqlite DB containing list of tables to migrate
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -D SOURCE_DB, --source-db SOURCE_DB
+                            Name for source database
+      -O SOURCE_HOST, --source-host SOURCE_HOST
+                            Name for source database
+      -U SOURCE_USER, --source-user SOURCE_USER
+                            Name for source database
+      -d TARGET_DB, --target-db TARGET_DB
+                            Name for target database
+      -o TARGET_HOST, --target-host TARGET_HOST
+                            Name for target database
+      -u TARGET_USER, --target-user TARGET_USER
+                            PG user to connect to target DB as. This user should
+                            be able to connect to the targetDB without a password.
+      --start-date START_DATE
+                            Only migrate tables with date on or after this date.
+                            Format YYYY-MM-DD
+      --end-date END_DATE   Only migrate tables with date before this date. Format
+                            YYYY-MM-DD
+      --start-after-table START_AFTER_TABLE
+                            Skip all tables up to and including this one
+      --table TABLE         Only migrate this table
+      --confirm             Confirm before each table.
+      --dry-run             Only output the commands.
+      --parallel PARALLEL   How many commands to run in parallel
+    ```
+
+    Notes:
+    * For best performance run this either on the source DB host or the target DB host. You can copy the
+      script to the host and it should not require any additional libraries to be installed other than python
+      itself.
+    * For authentication set up a [pgpass](https://www.postgresql.org/docs/9.6/libpq-pgpass.html) file
+      for the user running the script.
+    * Use `--parallel` to run multiple dumps in parallel
```

Comment:
should we include a note on metrics to check? For example we may want to make sure that we aren't saturating network or disk i/o